jsonnet/main.jsonnet

```diff
@@ -0,0 +1,160 @@
+                volumes: [
+                    {
+                        name: "benchmarks-runner",
+                        persistentVolumeClaim: {
+                            claimName: "monitoring-benchmarks-data-claim"
+                        },
+                    }
+                ],
+                containers: [
+                    {
+                        name: "runner",
+                        image: config['runner_image'],
+                        imagePullPolicy: 'Always',
+                        command: [ 'make'],
+                        args: ['run/benchmarks/continuously' ],
+                        volumeMounts: [
+                            {
+                                name: 'benchmarks-runner',
+                                mountPath: '/var/lib/benchmarks'
+                            }
+                        ],
+                        env: [
+                            {
+                                name: 'BENCHMARKS_RUNS_ROOT', 
+                                value: '/var/lib/benchmarks/runs'
+                            },
+                            {   
+                                name: 'PODS_PER_NODE',
+                                value: std.toString(config['pods_per_node'])
+                            },
+                            {   
+                                name: 'POD_CHURNING_PERIOD',
+                                value: config['pod_churning_period']
+                            },
+                            {   
+                                name: 'NUMBER_OF_NS',
+                                value: std.toString(config['number_of_ns'])
+                            },
+                        ],
+                        securityContext: {
+                            allowPrivilegeEscalation: false,
+                            capabilities: {
+                                drop: [ "ALL" ],
+                            },
+                            runAsNonRoot: true,
+                            seccompProfile: {
+                                type: 'RuntimeDefault',
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+    }
+};
+
+local benchmarks_runner_rbac() =
+[
+{
+    apiVersion: "rbac.authorization.k8s.io/v1",
+    kind: "ClusterRole",
+    metadata: {
+        name: "benchmarks-runner-role"
+    },
+    rules: [
+        {
+            apiGroups: ['*'], resources: ['*'], verbs: ['*'],
+        }
+    ]
+},
+{
+    apiVersion: "v1",
+    kind: "ServiceAccount",
+    metadata: {
+      name: "benchmarks-runner",
+      namespace: benchmarks_namespace
+    }
+},{
+    apiVersion: "rbac.authorization.k8s.io/v1",
+    kind: "ClusterRoleBinding",
+    metadata: {
+        name: "benchmarks-runner-rolebinding",
+    },
+    roleRef: {
+      apiGroup: "rbac.authorization.k8s.io",
+      kind: "ClusterRole",
+      name: "benchmarks-runner-role"
```

Comment:
Why not just give `cluster-admin` premissions?

Reply:
This way we can start with these crude permissions, and eventually tune them to have the minimal permissions that work. At this point I agree this is basically admin permissions, but it is not much code anyway, and it already sets the stage for restricting permissions later